micro-ui/web/micro-ui-internals/packages/modules/core/CHANGELOG.md

```
@@ -1,26 +1,31 @@
 # Changelog 
 
-## [1.8.42]  [18-Jun-2025]
--Updated the loader logic in login page
 
-## [1.8.41]  [12-Jun-2025]
--Introduced new hook `useLoginConfig` for mdms call
+## [1.8.43]  [26-June-2025]
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)